### PR TITLE
Sso

### PR DESCRIPTION
**Pull request by dittonjs**
so sso!!!!

### CODE REVIEW

File: app/controllers/proctored_exams_controller.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def start_quiz
     quiz = HTTParty.get(
       "#{plugin.settings[:adhesion_url]}/api/proctored_exams?#{query}",
       headers: headers,
-      # verify: false,
+      verify: false,
```
Comment (bfcoder) [inline app/controllers/proctored_exams_controller.rb]: Delete me

Comment (jbasdf) [reply app/controllers/proctored_exams_controller.rb]: Is this not needed?

Comment (bfcoder) [reply app/controllers/proctored_exams_controller.rb]: No, `verify: false` means "Don't use SSL"

Comment (bfcoder) [reply app/controllers/proctored_exams_controller.rb]: So at a minimum it needs to be commented out again.

Comment (bfcoder) [reply app/controllers/proctored_exams_controller.rb]: It is only there for development.

Comment (jbasdf) [reply app/controllers/proctored_exams_controller.rb]: Ok. Joseph can you comment it out?

**Review by jbasdf — APPROVED** [review]
LGTM

**Pull request merged**